devlab/tests/generate_load.py

```diff
@@ -557,7 +557,53 @@ def create_routers(self):
         for router in self.config.routers:
             self.neutronclient.create_router(router)
 
-    @clean_if_exists
+    def get_subnet_id(self, name):
+        subs = self.neutronclient.list_subnets()['subnets']
+        for sub in subs:
+            if sub['name'] == name:
+                return sub['id']
+        return None
+
+    def get_pool_id(self, name):
+        pools = self.neutronclient.list_pools()['pools']
+        for pool in pools:
+            if pool['name'] == name:
```

Comment:
Neutron allows to have more than one LB pool with the same name, this needs to be refactored to account for that fact.